lib/mumukit/login/helpers/login_controller_helpers.rb

```diff
@@ -1,7 +1,7 @@
 module Mumukit::Login::LoginControllerHelpers
 
   def login_current_user!
-    mumukit_controller.write_cookie!(:login_organization, organization_name)
+    mumukit_controller.mucookie.write!(:login_organization, organization_name)
```

Comment:
Is this change necessary?

Reply:
Well... as I said the other day, our login seems to be broken in iframes because it fails to access sessionStorage so I guess it wouldn't make a difference either way ¯\\\_(ツ)_/¯.

That being said without this change this cookie would be set with no SameSite attribute, which would make it default to Lax and not be sent in iframes which _could_ break login if it had been working in the first place.